ntc_templates/templates/arista_eos_show_ip_bgp_detail.textfsm

```diff
@@ -0,0 +1,42 @@
+Value LOCAL_PREF (\d+)
+Value WEIGHT (\S+)
+Value ORIGIN (.+)
+Value Filldown NETWORK (\S+)
```

Comment:
@tom0010 can we break out network into PREFIX and PREFIX_LENGTH? This is an updated data model that I need to add a PR shortly to add in.

Reply:
> 

@jvanderaa How can we do this in the case of a Filldown? What if the prefix is matched but not the subnet and vice versa? This may give us the wrong values.

Reply:
@jmcgill298 do you have any ideas on how to do a filldown where we want to capture two things?

Reply:
https://github.com/tom0010/ntc-templates/pull/1

Reply:
Thanks @jmcgill298.
How does it stop the filldown? I can see in the tests it does do that, and I've merged. Just wondering how for future.